src/utils.ts

```diff
@@ -198,6 +198,13 @@ export function getPropertyAccessor(objectName: string, propertyName: string, op
     : `${objectName}${optional ? '?.' : ''}[${JSON.stringify(propertyName)}]`;
 }
 
+export function impSlash(options: Options, spec: string) {
+  if (!spec.includes('/'))
```

Comment:
There are a lot of imports like `@foo/bar` which have a slash but are not referencing a specific file.

I don't see why it's necessary to add another function `impSlash` as opposed to just appending `importSuffix` everywhere `protobufjs/minimal` is imported?

Reply:
That's what the function it's doing. I just thought it looks cleaner to have it as util. What might be redundant are the includes check.

Reply:
Maybe call it `impFile` instead of `impSlash` and remove the `/` check ?

Reply:
That sounds great. 👍